src/main/java/io/confluent/maven/resolver/ResolveVersionRangeMojo.java

```diff
@@ -87,6 +87,9 @@
 	@Component
 	private RepositorySystem repoSystem;
 
+	@Parameter(property = "resolve-range.skip", defaultValue = "true")
+	private boolean skip;
```

Comment:
instead of making this change can we just delete this class entirely? upstream hasn't changed in years, so I don't think trying to track upstream changes is going to be an issue -- we probably just want to take on all maintenance of this ourselves moving forward.